openpype/hosts/maya/api/pipeline.py

```diff
@@ -627,6 +627,14 @@ def on_new():
         cmds.evalDeferred(
             "from openpype.hosts.maya.api import lib;"
             "lib.add_render_layer_change_observer()")
+        autobuild_first_workfile = """
+from openpype.pipeline.workfile import workfile_template_builder
+from openpype.hosts.maya.api.workfile_template_builder import build_workfile_template  # noqa
+builder = workfile_template_builder.build_first_workfile_from_template_builder()  # noqa
+if builder:
+    build_workfile_template()
+"""
+        cmds.evalDeferred(autobuild_first_workfile)
```

Comment:
Oh boy. :) 

Let's expose a method directly that just does:
```
def auto_build_first_workfile():
    if should_build_first_workfile():
        build_workfile_template()
```
☝️ And that function I suspect should be in the global pipeline workfile.

And for readability just trigger it directly through `maya.utils.executeDeferred(auto_build_first_workfile)` so you don't need to work with re-importing and the whole messy string management. A similar thing is done [here](https://github.com/ynput/OpenPype/pull/5226/files#diff-8481a901810ec3f718a093664b6877423345730dc3719122846c4c83a9ff34d5R560-R572).

Reply:
Thank you for your advice, I was kind of stuck on this part. I knew this was a little bit messed up, but I didn't found an other way to do it :)